packages/hash/api/src/graphql/resolvers/knowledge/model-mapping.ts

```diff
@@ -24,56 +24,56 @@ export const mapEntityModelToGQL = (
   accountId: entityModel.ownedById,
   properties: entityModel.properties,
   /**
-   * To be used by the `KnowledgeEntity` `__resolveType` resolver method to reliably determine
+   * To be used by the `PersistedEntity` `__resolveType` resolver method to reliably determine
    * the GQL type of this entity. Note that this is not exposed in the GQL type definitions,
    * and is therefore not returned to GraphQL clients.
    */
   workspaceTypeName: entityModel.entityTypeModel.workspaceTypeName,
 });
 
-export type ExternalKnowledgePageResolversGQL =
-  | ExternalKnowledgeEntityResolversGQL
+export type ExternalPersistedPageResolversGQL =
+  | ExternalPersistedEntityResolversGQL
   | "contents";
-export type UnresolvedKnowledgePageGQL = Omit<
-  KnowledgePage,
-  ExternalKnowledgePageResolversGQL
+export type UnresolvedPersistedPageGQL = Omit<
+  PersistedPage,
+  ExternalPersistedPageResolversGQL
 >;
 
 export const mapPageModelToGQL = (
   pageModel: PageModel,
-): UnresolvedKnowledgePageGQL => ({
+): UnresolvedPersistedPageGQL => ({
   ...mapEntityModelToGQL(pageModel),
   title: pageModel.getTitle(),
   properties: pageModel.properties,
   archived: pageModel.getArchived(),
   summary: pageModel.getSummary(),
 });
 
-export type ExternalKnowledgeBlockResolversGQL =
-  | ExternalKnowledgeEntityResolversGQL
+export type ExternalPersistedBlockResolversGQL =
```

Comment:
I agree with you about the casing consistency. I guess his would be a good place to do the `GQL` => `Gql` rename